fastlane/swift/Runner.swift

```diff
@@ -60,12 +60,12 @@ class Runner {
 
 // Handle threading stuff
 extension Runner {
-    func startSocketThread() {
+    func startSocketThread(port: UInt32 = 2000) {
```

Comment:
Is there a way to remove this default to `2000` if its being defaulted in `ArgumentProcessor.swift` already?

Reply:
Yes! You're right.